core/src/migrations/000100-removeImportData.ts

```diff
@@ -0,0 +1,60 @@
+import Sequelize from "sequelize";
+
+export default {
+  up: async (
+    queryInterface: Sequelize.QueryInterface,
+    DataTypes: typeof Sequelize
+  ) => {
+    await queryInterface.removeColumn("imports", "rawData");
+    await queryInterface.removeColumn("imports", "oldRecordProperties");
+    await queryInterface.removeColumn("imports", "newRecordProperties");
+    await queryInterface.removeColumn("imports", "oldGroupIds");
+    await queryInterface.removeColumn("imports", "newGroupIds");
+    await queryInterface.removeColumn("imports", "processedAt");
+
+    await queryInterface.sequelize.query(
+      `UPDATE imports SET state='importing' where state='processing'`
+    );
+
+    await queryInterface.addColumn("records", "readyToExport", {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+      defaultValue: false,
+    });
+
+    await queryInterface.changeColumn("records", "readyToExport", {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+    });
```

Comment:
Should probably make the new readyToExport column first, and populate the initial values from the old import columns - readyToExport=true if processedAt IS NULL